beamer/tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+from ape.api.providers import Web3Provider
+
+
+Web3Provider.send_call = Web3Provider._send_call  # type: ignore
```

Comment:
We should add a comment in here as to why we're doing this. E.g.
```
# Make ape do the call directly, instead of using chain_manager.isolate() to
# do a snapshot, make a transaction and then revert to the original chain state.
```